src/common/logger/logger_settings.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include <string>
+
+struct LogSettings
+{
+    inline const static std::wstring defaultLogLevel = L"warn";
```

Comment:
let's also add a comment that these strings are not localizable

Reply:
added